shared/bin/pacific/setup-ceph.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+cd /ceph
+python3 -m pip install --upgrade pip
+
+find . -name \*.pyc -delete
+./install-deps.sh
+
+ARGS="-DENABLE_GIT_VERSION=OFF -DWITH_TESTS=ON -DWITH_CCACHE=ON $ARGS"
+ARGS="-DWITH_PYTHON3=3 -DWITH_RADOSGW_AMQP_ENDPOINT=OFF -DWITH_RADOSGW_KAFKA_ENDPOINT=OFF $ARGS"
+
+NPROC=${NPROC:-$(nproc --ignore=2)}
+
+# Other dependencies
+zypper -n install utf8proc-devel
+
+# SSO dependencies
+zypper -n install libxmlsec1-1 libxmlsec1-nss1 libxmlsec1-openssl1 xmlsec1-devel xmlsec1-openssl-devel
+pip install python3-saml
+
+if [ "$CLEAN" == "true" ]; then
+    echo "CLEAN INSTALL"
+    git clean -fdx
+fi
+
+git config --global --add safe.directory '*'
```

Comment:
I think this should be done on your local git repository and does not need to be done here in this script.

Reply:
I'm not completely sure about this. I did configure it on my local git repository. Nevertheless I received:

```
fatal: detected dubious ownership in repository at '/ceph/ceph-erasure-code-corpus'
To add an exception for this directory, call:

	git config --global --add safe.directory /ceph/ceph-erasure-code-corpus
fatal: Unable to find current revision in submodule path 'ceph-erasure-code-corpus'
```

while running `setup-ceph.sh` on my pacific container :thinking:

Reply:
Yes, that line `git config --global --add safe.directory '*'` not resolved the issue actually.